src/parser/mod.rs

```diff
@@ -9092,6 +9092,16 @@ impl<'a> Parser<'a> {
                     });
                 }
             }
+            if self.dialect.supports_group_by_special_grouping_sets()
+                && self.parse_keywords(&[Keyword::GROUPING, Keyword::SETS])
+            {
+                self.expect_token(&Token::LParen)?;
+                let result = self.parse_comma_separated(|p| p.parse_tuple(true, true))?;
```

Comment:
should this be comma separate expr or similar instead of tuple?
here's one example from the [hive doc](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=30151323#EnhancedAggregation,Cube,GroupingandRollup-GROUPINGSETSclause) that we seem to not be able to support as  a result of using parse_tuple
```sql
SELECT a, b, SUM( c ) FROM tab1 GROUP BY a, b GROUPING SETS ( (a, b), a, b, ( ) )
```